kernel/safe_typing.mli

```diff
@@ -169,6 +169,12 @@ val add_module_parameter :
   MBId.t -> Entries.module_struct_entry -> Declarations.inline ->
     Mod_subst.delta_resolver safe_transformer
 
+(** returns the number of module parameters *)
```

Comment:
Should state whether it's the parameters of the topmost module or all of them. eg what is returned after
~~~coq
Module F(A:T).
  Module Inner.
~~~
?
What about
~~~coq
Module F(A:T).
  Module F'(B:TT).
~~~
?

Reply:
Thkns, I need  to get the full list to know if I'm a functor.